src/Lykke.SettingsReader/SettingsTemplate/JsonSettingsTemplateGenerator.cs

```diff
@@ -0,0 +1,25 @@
+using System.Text.Json;
+using Microsoft.Extensions.Configuration;
+
+namespace Lykke.SettingsReader.SettingsTemplate;
+
+internal class JsonSettingsTemplateGenerator : IJsonSettingsTemplateGenerator
+{
+    private readonly IConfiguration _configuration;
+
+    public JsonSettingsTemplateGenerator(IConfiguration configuration)
+    {
+        _configuration = configuration;
```

Comment:
I think it is vital to have the instance of `IConfiguration`:
```suggestion
        _configuration = configuration ?? throw new ArgumentNullException(...);
```